pylint/checkers/strings.py

```diff
@@ -353,7 +353,8 @@ def visit_binop(self, node):
                     arg_type = utils.safe_infer(arg)
                     if (
                         format_type is not None
-                        and arg_type not in (None, astroid.Uninferable)
+                        and arg_type
+                        and not arg_type == astroid.Uninferable
```

Comment:
```suggestion
                        and arg_type != astroid.Uninferable
```
Don't know, maybe `is not` will work too.